docs/user_guide/use_case/multi_gpu_training.md

```diff
@@ -56,5 +63,5 @@ model.train(
 
 3. There are a few caveats with the current implementation:
    - During interactive session, like in jupyter notebook, we can only train 1 model in multi GPU mode, per session.
-   It means that we cant train SCANVI model from SCVI model, if the SCVI model was trained in the same notebook. Therefore, need to train and save the SCVI model in another session and load it in the other session.
-   - It cant run with early stopping right now (and some models, like totalvi, run it by default), so we disable early stopping once running with DDP. the reason is that validation loop should be running on 1 device only and not multiGPU.
+   It means that we cant train SCANVI model from SCVI model, if the SCVI model was trained in the same notebook. Therefore, need to train and save the SCVI model in another session and load it in the other session. This is a torch lightning caveat.
+   - It cant run with early stopping right now (and some models, like totalvi, use early stopping by default), so we disable early stopping once running with DDP. the reason is that validation loop should be running on 1 device only and not multiGPU.
```

Comment:
we should likely fix this behavior in the near future and note it as a to-do (after custom dataloader).